### PR TITLE
[37037] Fix bim seeder overriding existing theme

### PR DESCRIPTION
**Pull request by oliverguenther**
https://community.openproject.org/wp/37037

### CODE REVIEW

File: modules/bim/app/seeders/bim/basic_data/theme_seeder.rb [hunk 1]
```diff
@@ -39,6 +39,10 @@ def seed_data!
           .new(theme)
           .call
       end
+
+      def applicable?
+        !DesignColor.any?
```
Comment (wielinde) [inline modules/bim/app/seeders/bim/basic_data/theme_seeder.rb:44]:
```suggestion
        DesignColor.empty?
```

Comment (wielinde) [reply modules/bim/app/seeders/bim/basic_data/theme_seeder.rb:44]: IMHO `empty?` is easier to read than the negation bang way.

https://api.rubyonrails.org/v4.1.0/classes/ActiveRecord/Relation.html#method-i-empty-3F

**Review by wielinde — APPROVED** [review]
LGTM. Check my tiny remark and feel free to merge.

File: modules/bim/app/seeders/bim/basic_data/theme_seeder.rb (hunk 1 above)
Comment (oliverguenther) [reply modules/bim/app/seeders/bim/basic_data/theme_seeder.rb:44]: Yes, I agree. Will change that

**Pull request merged**